src/styles/core/typography/_font-styles.scss

```diff
@@ -153,7 +153,7 @@ $font-styles: (
   documentation-code-listing-number: (
     large: 12_18_normal_compact_mono,
   ),
-  documentation-figcaption: (
+  documentation-caption: (
```

Comment:
I think I might have missed this in the original PR too, but you'll also need to update the place where this is being used with the old name in `Caption`